CHANGELOG.md

```diff
@@ -10,9 +10,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [v0.18.8] - 2024-11-22
 
 ### Added
+
```

Comment:
Unrelated to this bug, but is a linting issue on `main` that is causing checks to fail